### PR TITLE
Add template literals revision upgrade post

### PR DESCRIPTION
**Pull request by xtuc**
Part of #1209

### CODE REVIEW

**Comment by hzoo** [comment]
👍 although now I realize it doesn't technically make sense as part of the es2015 plugin. It doesn't matter for the purposes that we want everyone to use preset-env though. What should we do here?

File: _posts/2017-03-01-upgrade-to-babel-7.md [hunk 1]
```diff
@@ -230,6 +230,26 @@ let p2 = Point.secondConstructor(3, 4);
...
+
+>  Template Literals Revision updated [#5523](https://github.com/babel/babel/pull/5523) ![low](https://img.shields.io/badge/risk%20of%20breakage%3F-low-yellowgreen.svg)
+
+See the proposal for [Template Literals Revision](https://tc39.github.io/proposal-template-literal-revision/).
+
+```js
+tag`\unicode and \u{55}`;
+```
+
+It cause Babel 6 to throw `Bad character escape sequence (5:6)`.
```
Comment (existentialism) [inline _posts/2017-03-01-upgrade-to-babel-7.md]: Maybe move before the code block,

> The following code in Babel 6 will throw `Bad character escape sequence (5:6)`:

File: _posts/2017-03-01-upgrade-to-babel-7.md [hunk 2]
```diff
@@ -230,6 +230,26 @@ let p2 = Point.secondConstructor(3, 4);
...
+
+This has been fixed in Babel 7 and generate something like the following:
```
Comment (existentialism) [inline _posts/2017-03-01-upgrade-to-babel-7.md]: generate -> generates

**Comment by xtuc** [comment]
@existentialism thanks, I made the requested change

**Pull request merged**